ja/lessons/advanced/otp-distribution.md

```diff
@@ -0,0 +1,486 @@
+---
+version: 1.0.0
+title: OTPディストリビューション
+---
+
+## ディストリビューションへの入門
+Elixirアプリは、単体のホストや複数のホストにまたがって分散された(distributed)一連の異なるノードで実行することができます。
+Elixirは、このレッスンで概説するいくつかの異なるメカニズムを通して、それらのノード間で通信をすることが可能です。
+
+{% include toc.html %}
+
+## ノード間通信
+
+ElixirはErlang VMで動作しますが、これはErlangの強力な [分散機能](http://erlang.org/doc/reference_manual/distributed.html) へアクセス可能であることを意味します。
+
+> 分散されたErlangシステムは互いに通信するいくつものErlangランタイムシステムを含んでいます。
+これらのそれぞれのランタイムはノードと呼ばれます。
+
+ノードは名前が与えられた任意のErlangランタイムシステムです。
+`iex` セッションを起動して名前を与えることで、ノードを開始できます:
+
+```bash
+iex --sname alex@localhost
+iex(alex@localhost)>
+```
+
+別のターミナルウィンドウで、もう一つのノードを起動してみましょう:
+
+```bash
+iex --sname kate@localhost
+iex(kate@localhost)>
+```
+
+これら2つのノードは `Node.spawn_link/1` を使うことでもう一方にメッセージを送ることができます。
+
+### `Node.spawn_link/1` による通信
```

Comment:
isn't it `Node.spawn_link/2`? https://hexdocs.pm/elixir/Node.html#spawn_link/2

Reply:
It seems original text is wrong..
https://github.com/elixirschool/elixirschool/pull/1841